jolly-jellyfish/src/page_maker/tests.py

```diff
@@ -38,13 +50,115 @@ def test_register_and_home(self):
         self.assertNotIn(USERNAME.encode(), res, 'Our username was already in the home view.')
 
 
+class UserViewsTest(TestCase):
+    fixtures = ['users.yaml', 'webpages.yaml', 'templates.yaml']
+
+    def test_user_detail(self):
+        user = User.objects.get(pk=1)
+        res = self.client.get('/users/1/')
+        self.assertEqual(res.status_code, 200)
+        self.assertEqual(res.context['viewed_user'], user)
+        self.assertIsInstance(res.context['user_pages'][0], Webpage)
+
+    def test_user_update(self):
```

Comment:
Should we test that a non-signed-in user cannot update a user? That sounds like some logic we want to make sure works.